src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -405,7 +411,12 @@ int ITupleInternal.GetHashCode(IEqualityComparer comparer)
         /// </remarks>
         public override string ToString()
         {
-            return "(" + Item1 + ")";
+            return "(" + Item1.ToString() + ")";
```

Comment:
nit: string interpolation over string concatenation.

```c#
return $"({Item1})";  // no need to call ToString()
```

(same goes for the rest of the `return` statements in this file where strings are explicitly concatenated)